commit b997d8e7936de46f4f78a49dc9ae00db46c9e803
Author: Jan Beich <jbeich@FreeBSD.org>
Date:   Thu Apr 15 20:49:24 2021 +0000

    meson: unbreak with system wlroots after fd4ea3542fdc
    
    meson.build:50:2: ERROR: All keyword arguments must be after positional arguments.

diff --git a/meson.build b/meson.build
index a2b86db..2345fc7 100644
--- a/meson.build
+++ b/meson.build
@@ -47,7 +47,7 @@ if wlroots_proj.found()
   wlroots_conf = wlroots_proj.get_variable('conf_data')
   wlroots_has_xwayland = wlroots_conf.get('WLR_HAS_XWAYLAND') == 1
 else
-  wlroots       = dependency('wlroots', version: '>=0.13.0', '<0.14.0')
+  wlroots       = dependency('wlroots', version: ['>=0.13.0', '<0.14.0'])
   wlroots_has_xwayland = cc.get_define('WLR_HAS_XWAYLAND', prefix: '#include <wlr/config.h>', dependencies: wlroots) == '1'
 endif
 wayland_server  = dependency('wayland-server', version: '>=0.19.0')
